### PR TITLE
DRILL-3724 - added javadoc for core classes for storage plugin development.

### PR DESCRIPTION
**Pull request by ebegoli**
Changes are only to the javadoc. Code is not affected.

### CODE REVIEW

File: exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/easy/EasyFormatPlugin.java [hunk 1]
```diff
@@ -115,6 +115,10 @@ public boolean isBlockSplittable() {
     return blockSplittable;
   }
 
+  /** Whether or not this format could also be in a compression container (for example: csv.gz versus csv).  
+   * This is an external compression container that would be handled outside of the format code. 
+   * This is an internal compression scheme, not supported by Parquet.
```
Comment (jacques-n) [inline exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/easy/EasyFormatPlugin.java]: A little unclear.  Maybe "If this format uses its own internal compression scheme, such as Parquet does, then this should return false."

**Comment by ebegoli** [comment]
Fixed. Review please.

**Comment by jacques-n** [comment]
Looks good to me +1.  I'll get this merged

**Comment by jacques-n** [comment]
This has been merged. However, I forgot to include the "merged in" message so @ebegoli , can you please close?

**Comment by ebegoli** [comment]
Closing.

**Pull request closed without merging**